### PR TITLE
Use same lifespan for cookies as for tokens

### PR DESCRIPTION
**Pull request by alinhle**
## What does this PR do?
- Uses the same lifespan for safari login cookies as for jwt tokens

## How do I test this PR?

- Login with Safari
- Inspect cookie in `storage` tab and check the expiry time to match the JWT token expiry time.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -92,6 +92,7 @@
     "minimist": "^1.2.0",
     "mongoose": "^4.9.8",
     "morgan": "^1.8.1",
+    "ms": "^2.0.0",
```
Comment (wyattjoh) [inline package.json:95]: We already have the https://www.npmjs.com/package/parse-duration package included, no need to import this new package.

**Review by wyattjoh — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (alinhle) [reply package.json:95]: This is the same package used by the JWT token generator. It ensures that the same logic is applied when parsing the duration. There are some subtle parsing differences with `parse-duration` and the `ms`.

Comment (alinhle) [reply package.json:95]: We can either preparse the duration using `parse-duration` before passing it to the JWT Token Generator. Or port our codebase to `ms`.

Comment (alinhle) [reply package.json:95]: Which one do you prefer?

**Review by wyattjoh — APPROVED** [review]

**Pull request merged**